modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger_dev.json

```diff
@@ -2059,6 +2059,19 @@
                             "ROC",
                             "ROK",
                             "Singapore",
+                            "SystemV/AST4",
+                            "SystemV/AST4ADT",
+                            "SystemV/CST6",
+                            "SystemV/CST6CDT",
+                            "SystemV/EST5",
+                            "SystemV/EST5EDT",
+                            "SystemV/HST10",
+                            "SystemV/MST7",
+                            "SystemV/MST7MDT",
+                            "SystemV/PST8",
+                            "SystemV/PST8PDT",
+                            "SystemV/YST9",
+                            "SystemV/YST9YDT",
```

Comment:
These `SystemV/*` timezones were the only ones we seemed to be missing.

I was torn about whether to filter these out - they do refer to valid IANA timezones (for example, `SystemV/PST8` actually refers to Pacific Standard Time), but I can't imagine them being used much by our consumers. I can go ahead and hard-code a filter to remove any `SystemV/*` timezone if people prefer - it would be pretty easy.

Reply:
I think it's probably fine to leave them in.